episodes/02-image-basics.md

```diff
@@ -8,11 +8,11 @@ exercises: 5
 
 - Define the terms bit, byte, kilobyte, megabyte, etc.
 - Explain how a digital image is composed of pixels.
-- Recommend using imageio (resp. skimage) for I/O (resp. image processing) tasks.
+- Recommend using imageio (resp. scikit-image) for I/O (resp. image processing) tasks.
```

Comment:
I'm surprised by this change because this is the version we already have on `main`:
https://github.com/datacarpentry/image-processing/blob/893e6e6810267cd11d25e809721da9c9258dbbb6/episodes/02-image-basics.md?plain=1#L11